features/cells/src/main/java/com/wire/android/feature/cells/ui/CellScreenContent.kt

```diff
@@ -65,7 +65,7 @@ internal fun CellScreenContent(
     onFolderClick: (CellNodeUi.Folder) -> Unit,
     downloadFileState: StateFlow<CellNodeUi.File?>,
     menuState: Flow<MenuOptions?>,
-    showPublicLinkScreen: (String, String, String?) -> Unit,
+    showPublicLinkScreen: (String, String, String?, Boolean) -> Unit,
```

Comment:
suggestion: with that many parameters I would create a dedicated data class or at least put some names so that it's clear what each parameter is for